content/60.methods.md

```diff
@@ -182,106 +193,132 @@ Manubot users can write in the simple markdown format, render the manuscript in
+In the future, it may be possible to streamline this process through integration with a tool such as CoronaCentral [@doi:10.1073/pnas.2100766118] to automatically identify relevant, high-impact papers that need to be included, although expertise would still be required to incorporate them.
+Another challenge involves tracking preprints as they are reviewed or critiqued, revised, and potentially published.
+While updating the content of the manuscript would likely fall to human contributors, automatic detection of published versions of preprints could be integrated in the future (as developed by [@doi:10.1101/2021.03.04.433874]).
+These challenges are exacerbated by the scale of the infodemic, but developing solutions would benefit future projects with more typical trends in literature.
+Similarly, outputting machine readable summaries of key information in the manuscripts developed by the Consortium could reduce our contribution to the infodemic.
+As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction, as it could be used to extract relationships among clinical trials, genes, publications, and other entities.
+Thus, the experience of using Manubot for a massive non-technical project has laid the foundation for future additions to enhance user experience and inclusivity.
+
+### CONCLUSION 
 
 With the worldwide scientific community uniting during 2020 and 2021 to investigate SARS-CoV-2 and COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
 As many other efforts have described, the publishing rate of formal manuscripts and preprints about COVID-19 has been unprecedented [@doi:10.1053/j.ackd.2020.08.003], and efforts to review the body of COVID-19 literature are faced with an ever-expanding corpus to evaluate.
 In the case of the seven manuscripts produced by the COVID-19 Review Consortium, Manubot will allow for continuous updating of the manuscripts as the pandemic enters its second year and the landscape shifts with the emergence of promising therapeutics and vaccines [@individual-pharmaceuticals; @individual-vaccines].
-These manuscripts pull data from four external sources, allowing for information and visualizations to be updated daily using CI.
+These manuscripts pull data from external sources, allowing for information and visualizations to be updated daily using CI.
 This computational approach allows for some of the updating process to be off-loaded so that domain experts can focus on the broader implications of new information as it emerges.
 As a result, centralizing, summarizing, and critiquing data and literature broadly relevant to COVID-19 can help to expedite the interdisciplinary scientific process that is currently happening at an advanced pace.
+As of September 13, 2021, almost 3,000 commits have been made to the manuscript across 575 merged pull requests.
```

Comment:
Woah!  Where did the 3,000 commits come from?  Is this one of the stats you now track in a JSON file?  It looks like it distinguishes between commits associated with pull requests versus all 5,000 commits to master (which includes rootstock and non-content commits).

Reply:
Never mind, now I see `"manuscript_commits": 2886` in `covid19-review-stats.json`